scripts/apps/users/directives/UserEditDirective.ts

```diff
@@ -10,6 +10,10 @@ import {noop} from 'lodash';
 // according to the latest data on the server
 let clearOrigUserWatcher = noop;
 
+// only initialize after selecting a user for editing/creation
+// having it running when switcing between users can cause fields to be modified or produce an incorrect dirtiness value
+let clearUserWatcher = noop;
```

Comment:
I think it would be better to have this in the link func, otherwise scope and element would probably stay around even when that directive is destroyed